### DISCUSSION TITLE
启动器无法正常运行

### DISCUSSION THREAD

**Discussion by msttyy** [natfrp/wiki#248] [使用问题] [answered]
版本:v2.0.5.6
日志截图:
![image](https://user-images.githubusercontent.com/15191671/210059290-e90c16d6-d779-4b9d-b9f1-d784068179f7.png)

**Comment by fakeboboliu** [comment, marked as answer]
您好，通常来讲，我们认为在出现此错误时应先检查网络，有下面几种原因可能导致此问题：

|问题|排查方法|解决方案|
|--|--|--|
| **网游**加速器类网络软件的历史遗留影响|用过加速器|打开加速器软件，使用内置的重置 LSP 或重置 Winsock 功能|
|运营商 DNS 对冷门网站歧视问题|执行 `nslookup api.natfrp.com` 无应答字样，但 `nslookup api.natfrp.com 223.5.5.5` 有|更换系统 DNS|
|本机网络配置有误（如跃点、路由表等）|在终端中执行 `nslookup api.natfrp.com` 无法看到应答字样，同时您很信任自己的运营商不会进行奇怪行为|默认配置通常为正确，既然您有配错的能力那您应当有配回来的自信|
|运营商屏蔽我们的服务|以上排查均不符合，同时您对自己的网络提供商没有信任|算逑|
